apps/persons/factories.py

```diff
@@ -0,0 +1,47 @@
+"""
+Persons factories
+"""
+from cms.api import create_page
+import factory
+
+from .models import Person, PersonTitle
+
+
+class PersonTitleFactory(factory.django.DjangoModelFactory):
+    """
+    Person's PersonTitle factory
+    """
+
+    class Meta:
+        model = PersonTitle
+
+    title = factory.Faker("prefix")
+    abbreviation = factory.LazyAttribute(lambda o: o.title)
+
+
+class PersonFactory(factory.django.DjangoModelFactory):
+    """
+    Person factory
+    """
+
+    class Meta:
+        model = Person
+
+    first_name = factory.Faker("first_name")
+    last_name = factory.Faker("last_name")
+    person_title = factory.SubFactory(PersonTitleFactory)
+
+    @factory.lazy_attribute
+    def extended_object(self):
+        """
+        Automatically create a related page with a random title
```

Comment:
... with the person's name as title

otherwise we are confusing title and persontitle...